cite-gradle-plugin/src/main/kotlin/com/jakewharton/cite/plugin/gradle/CitePlugin.kt

```diff
@@ -30,13 +31,26 @@ public class CitePlugin : KotlinCompilerPluginSupportPlugin {
 			kotlin.sourceSets.getByName(MAIN_SOURCE_SET_NAME).dependencies {
 				compileOnly(target.citeApiDependency())
 			}
+			kotlin.sourceSets.getByName(TEST_SOURCE_SET_NAME).dependencies {
+				compileOnly(target.citeApiDependency())
+			}
 		}
 		target.plugins.withId("org.jetbrains.kotlin.android") {
 			applied = true
 			val kotlin = target.extensions.getByType(KotlinAndroidProjectExtension::class.java)
 			kotlin.sourceSets.getByName(MAIN_SOURCE_SET_NAME).dependencies {
 				compileOnly(target.citeApiDependency())
 			}
+			kotlin.sourceSets.getByName(TEST_SOURCE_SET_NAME).dependencies {
+				compileOnly(target.citeApiDependency())
+			}
+			target.configurations.names.forEach { configName ->
+				if (configName.contains("androidTest")) {
```

Comment:
This seems like it adds it to every test configuration rather than just `androidTestCompileOnly`. It probably means it gets added to the runtime dependencies as a result.

I'll confirm and switch it with the rebase + test updates.

Reply:
good point.